interface/patient_file/ccr_import.php

```diff
@@ -24,25 +24,22 @@
 <head>
 <?php Header::setupHeader(); ?>
 <title><?php echo xlt('Import');?></title>
-<style>
-.list-group-item {
-    display: list-item;
-}
-</style>
 </head>
-<body class="body_top" >
-  <main class="container">
+<body>
+  <div class="container mt-3">
     <div class="row">
-      <div class="col-md-6 offset-md-3">
-        <h3><?php echo xlt("Steps for uploading CCR XML");?></h3>
-        <ol class="list-group">
-          <li class="list-group-item"><?php echo xlt('For an existing patient, go to Patient Summary->Documents. For a new patient, go to Miscellanous->New Documents') . '.'; ?></li>
-          <li class="list-group-item"><?php echo xlt('Upload the xml file under the category CCR') . '.'; ?></li>
-          <li class="list-group-item"><?php echo xlt('After Uploading, click "Import"') . '.'; ?></li>
-          <li class="list-group-item"><?php echo xlt('Approve the patient from Patient/Client->Import->Pending Approval') . '.'; ?></li>
-        </ol>
-      </div>
+        <div class="col-12">
+            <h2><?php echo xlt("Steps for uploading CCR XML");?></h2>
+        </div>
+        <div class="col-12 jumbotron py-4">
```

Comment:
We don't need any col-12's here or rows since we are already at 100% width!!

Reply:
Seems like u missed one on line 32

Reply:
Hey @tywrenn 

When I remove the `.col-12` it still has the same issue.

Before:
![image](https://user-images.githubusercontent.com/22230889/89753286-37c47800-db0a-11ea-9f2b-ecc40c8107b9.png)

After:
![image](https://user-images.githubusercontent.com/22230889/89753291-3f841c80-db0a-11ea-9c26-696f891d8ded.png)